plugins/woocommerce/tests/e2e-pw/tests/merchant/create-post.spec.js

```diff
@@ -0,0 +1,59 @@
+const { test, expect, request } = require( '@playwright/test' );
+const { admin } = require( '../../test-data/data' );
+
+const postTitle = `Post-${ new Date().getTime().toString() }`;
+
+test.describe( 'Can create a new post', () => {
+	test.use( { storageState: process.env.ADMINSTATE } );
+
+	test.afterAll( async ( { baseURL } ) => {
+		const base64auth = btoa( `${ admin.username }:${ admin.password }` );
+		const wpApi = await request.newContext( {
+			baseURL: `${ baseURL }/wp-json/wp/v2/`,
+			extraHTTPHeaders: {
+				Authorization: `Basic ${ base64auth }`,
+			},
+		} );
+		let response = await wpApi.get( `posts` );
+		const allPosts = await response.json();
+
+		await allPosts.forEach( async ( post ) => {
+			if ( post.title.rendered === postTitle ) {
+				response = await wpApi.delete( `posts/${ post.id }` );
+				expect( response.ok() ).toBeTruthy();
+			}
+		} );
+	} );
+
+	test( 'can create new post', async ( { page } ) => {
+		await page.goto( 'wp-admin/post-new.php', {
+			waitUntil: 'networkidle',
```

Comment:
It shouldn't be necessary to explicitly put in a wait here.  Playwright should do it automatically (unless you're seeing something off when trying to enter the title in the next step...)

Reply:
Thank you. I will test with that removed and report if everything works as expected.